src/js/components/forms/Dropdown.jsx

```diff
@@ -56,6 +56,18 @@ export class Dropdown extends React.Component {
         return this.state.selectedOption;
     }
 
+    selectOption(option) {
```

Comment:
So just got back to this one. I don't think now is the best time to begin these larger changes for two reasons:
- They will commit us to a path that maybe not everyone is onboard with: @scherler had some objections to namespacing in the Gitter room yesterday.
- It will change behavior in Dropdown and I don't want to introduce risk of regression into parameterized runs and the editor.

I'm going to remove `selectOption` from this PR so we don't expose a method that we plan to remove later. I can accomplish the same behavior via a single LOC inside of Git creation, document the hack, and then fix it once after we get everyone on the same page and do the implementation in a reasonable time frame.